.github/workflows/release_publish.yml

```diff
@@ -34,12 +34,3 @@ jobs:
           PGP_SECRET: ${{ secrets.PGP_SECRET }}
           SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
           SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
-
-      - name: Find WAR file
-        id: find_war
-        run: echo "WAR_PATH=$(find . -name '*.war' | head -n 1)" >> $GITHUB_ENV
-
-      - name: Upload WAR file to GitHub Release
-        run: gh release upload ${{ github.event.release.tag_name }} ${{ env.WAR_PATH }} --repo ${{ github.repository }}
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
why? I quite like the idea behind this

Reply:
Because there is no war file used anymore. The application is packaged as JAR file and the web server backend is included in there. Because the sprig boot app used war and needed Tomcat.